libcontainer/nsenter/cloned_binary.c

```diff
@@ -372,35 +372,35 @@ static int seal_execfd(int *fd, int fdtype)
 	switch (fdtype) {
 	case EFD_MEMFD:
 		return fcntl(*fd, F_ADD_SEALS, RUNC_MEMFD_SEALS);
-	case EFD_FILE: {
-		/* Need to re-open our pseudo-memfd as an O_PATH to avoid execve(2) giving -ETXTBSY. */
-		int newfd;
-		char fdpath[PATH_MAX] = {0};
+	case EFD_FILE:{
+			/* Need to re-open our pseudo-memfd as an O_PATH to avoid execve(2) giving -ETXTBSY. */
```

Comment:
This is the only line > 80-chars; looks like it does not reformat comments, so perhaps manually wrap this one;

```suggestion
			/*
			 * Need to re-open our pseudo-memfd as an O_PATH to avoid
			 * execve(2) giving -ETXTBSY.
			 */
```

Reply:
Currently the max width is set to 120 for indent (and it was that way since indent was introduced).

Oh, and yes, indent seem to not try to break very long comments (see my comments in PR while I tested whether the check works).

As this comments fits into 120 columns I'm not going to touch it.